lib/model/model_wrapped.dart

```diff
@@ -24,6 +24,8 @@ class CourseWrapped extends BaseWrapped {
 
   Course get course => _course;
 
+  String get id => _course.id ?? "Not found"; // TODO: maybe throw error here?
```

Comment:
I would not throw an error here, since it would be pretty messy to catch it inside the widgets. An id should not miss though from the API responses, it is very unlikely that we would encounter this situation.